packages/samples/react/src/components/Navigation.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { KolAccordion, KolTree, KolTreeItem } from '@public-ui/react';
+import { useMobile } from '../hooks/useMobile';
+import type { Routes, Route } from '../shares/types';
+import { useHref, useMatch, useResolvedPath } from 'react-router-dom';
+
+type NavigationProps = {
+	routes: Routes;
+};
+
+function ComponentNavContainer({ children }: { children?: React.ReactNode }): React.ReactNode {
+	const isMobile = useMobile();
+
+	return isMobile ? (
+		<KolAccordion _label="Alle Komponenten" class="mt">
```

Comment:
```suggestion
		<KolAccordion _label="All components" class="mt">
```

War vermutlich schon vorher falsch. Wäre super, wenn wir das kurz mit anpassen könnten!